app/src/main/res/layout/developer_options_activity.xml

```diff
@@ -0,0 +1,34 @@
+<layout>
+
+  <androidx.drawerlayout.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/developer_options_activity_drawer_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:orientation="vertical">
+
+      <include
+        android:id="@+id/developer_options_activity_toolbar"
+        layout="@layout/toolbar" />
+
+      <FrameLayout
+        android:id="@+id/developer_options_fragment_placeholder"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" />
+    </LinearLayout>
+
+    <fragment
+      android:id="@+id/developer_options_activity_fragment_navigation_drawer"
+      android:name="org.oppia.android.app.drawer.NavigationDrawerFragment"
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:layout_gravity="start"
+      app:layout="@layout/drawer_fragment"
+      tools:layout="@layout/drawer_fragment" />
+  </androidx.drawerlayout.widget.DrawerLayout>
+</layout>
```

Comment:
Add `6dp` shadow below toolbar. You can find this in `help_activity`

Reply:
Done